### ISSUE TITLE
Mobile items not all hidden to screen readers

### ISSUE THREAD

**Issue by lkiesow** [virtUOS/wolke#35]
You hide a lot of elements on mobile. However, some of the items are still visible to screen readers. Here, that probably doesn't make sense:

https://github.com/user-attachments/assets/cb6954af-e454-4958-94d3-e7658b94b9ab

*(Turn on audio)*

PS: In the video I say that  “If you hide things in the app, you should also always hide it from the screen reader”. Listening to this, I realized that this obviously isn't universally correct. You can have information only visible to screen readers and not visible otherwise. What I mean is that here we have information available to all on desktop mode we deliberately hide on mobile so that users have to deal with less information. Since we hide it for that reason, it should also be hidden from screen readers.
